### DISCUSSION TITLE
Include all for ETH, exclude some for TON (devices, dualdevices)

### DISCUSSION THREAD

**Discussion by SonOfStan1926** [Lolliedieb/lolMiner-releases#1212] [Q&A] [answered]
Based on my reading & tests, --devices determines GPUs to use in miner & --dualdevices determines what should only use the second connection (TON in my case) based on 1.44 tests.

I want to do ETH all & selective TON, but the switches to not seem to allow for this type of setup or am I misunderstanding the documentation?  If possible how?  If not, can this be added.  Reason is that TON really heats up a few of my GPUs if using dual & I want to be able to do ETH only in this case as of this point.

**Comment by cypher619** [comment, marked as answer]
--maxdualimpact does this not --dualdevices see **Guide Tuning Dual Mining** in the Wiki section

**Reply by SonOfStan1926** [reply]
Thank you!
